### ISSUE TITLE
[ebc] fizzbuzz.c.eir.ebc Failed

### ISSUE THREAD

**Issue by retrage** [retrage/elvm#1]
```
./runtest.sh out/fizzbuzz.c.eir.ebc.out ebcvm out/fizzbuzz.c.eir.ebc
exception MEMORY: out of memory
```

**Comment by retrage** [comment]
This is because of fail of ELVM internal stack pointer setup.
```
main:
        mov D, SP
        add D, -1
        store BP, D
```
The `mem[BP] = mem[0] = SP - 1 = 0 - 1 = 0xffffff`

**Comment by retrage** [comment]
Fixed with commit `2bc235dda231c5215f0080f0229590ea57dafbaa`
